### ISSUE TITLE
Code coverage related tweaks

### ISSUE THREAD

**Issue by sidkshatriya** [Lullabot/amp-library#81]
- [x] Add a basic `phpunit.xml` configuration file to whitelist the files we want to do code coverage on
- [x] Find out why `ParsedUrlSpecStylesheetErrorAdapter.php` does not get any code coverage statistics when it is very obviously being called

**Comment by sidkshatriya** [comment]
The issue why there was no code coverage for `ParsedUrlSpecStylesheetErrorAdapter.php` was frustrating to track down but has now been resolved!

Initially I thought it was due to bugs in `phpunit/php-code-coverage` or `phpunit/phpunit` so I tried various version combinations with no avail; recompiling xdebug etc. (xdebug provides the code coverage data). There was simply no code coverage information displayed for `ParsedUrlSpecStylesheetErrorAdapter.php`  but the class was definitely being called.

The answer is subtle. The class was called:
ParsedUrlSpec**StyleSheet**ErrorAdapter but its corresponding file was 
ParsedUrlSpec**Stylesheet**ErrorAdapter.php 

Notice the case difference. The library works fine on MacOS (with autoloading etc) because its a case insensitive file system.

But the `phpunit/php-code-coverage` takes case into account when calculating code coverage. On renaming the filename everything works fine as far as code coverage is concerned.

Incidentally, on Linux the file system is case _sensitive_ so this would lead to more fatal problem like the file not being found whenever that particular class is needed to be autoloaded. Indeed this is what happens when I checked today.

Anyways its been fixed (See PR #82). 

_Note:_ The file `ParsedUrlSpecStyleSheetErrorAdapter.php` is a file that was merged only yesterday into master so this subtle problem was not long lived anyways on the master branch (as far as linux is concerned; there was no problem for MacOS).

**Comment by sidkshatriya** [comment]
BTW you can run code coverage in the following fashion:

(Make sure you have the xdebug extension installed)

```
$ cd <amp-library-root-director>
$ vendor/bin/phpunit tests --coverage-html coverage
$ cd coverage
$ open index.html
```
